syncer/syncer.go

```diff
@@ -26,15 +27,17 @@ type layerFetcher interface {
 	PollLayerHash(ctx context.Context, layerID types.LayerID) chan layerfetcher.LayerHashResult
 	PollLayerBlocks(ctx context.Context, layerID types.LayerID, hashes map[types.Hash32][]peers.Peer) chan layerfetcher.LayerPromiseResult
 	GetEpochATXs(ctx context.Context, id types.EpochID) error
-	GetTortoiseBeacon(ctx context.Context, id types.EpochID) error
+	GetTortoiseBeacon(ctx context.Context, id types.EpochID) error // TODO(nkryuchkov): remove
```

Comment:
fetcher shouldn't contain these methods, as it is not related to any fetching of data, where is tbDB created? where is it used except for the fetcher?

Reply:
`tbDB` is created in `node.go`, do you think it should be passed to syncer?